samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,296 @@
+#. Respond to the request by sending two notification attributes: the title and the message.
+   Set the :guilabel:`Apple Data Source` value in the Server to ``00 01 02 03 04 01 03 00 6E 52 46 03 02 00 35 32``.
+#. The application will print the received data on UART.
+   Verify that the UART output is as follows::
+
+      Title: nRF
+      Message: 52
+
+#. Update the :guilabel:`Apple Data Source` value again with the app identifier ``00 03 00 63 6F 6D``.
+#. Verify that the notification is received and the UART output is as follows::
+
+      App Identifier: com
+
+Retrieve app attributes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With the app identifier, you can request attributes of the app that sent the notification.
+
+The following table shows the format of a request to retrieve app attributes:
+
+   +------------------+---------------+--------------------+
+   | Field            | Example value | Interpretation     |
+   +==================+===============+====================+
+   | Command ID       | 1             | Get app attributes |
+   +------------------+---------------+--------------------+
+   | App identifier   | 6D 6F 63 00   | "com" + '\0'       |
+   +------------------+---------------+--------------------+
+   | Attribute ID     | 0             | Display name       |
+   +------------------+---------------+--------------------+
+
+The following table shows the format of a response that contains the requested app attributes:
+
+   +------------------+---------------+--------------------+
+   | Field            | Example value | Interpretation     |
+   +==================+===============+====================+
+   | Command ID       | 1             | Get app attributes |
+   +------------------+---------------+--------------------+
+   | App identifier   | 6D 6F 63 00   | "com" + '\0'       |
+   +------------------+---------------+--------------------+
+   | Attribute ID     | 0             | Display name       |
+   +------------------+---------------+--------------------+
+   | Length           | 04 00         | 0x0004             |
+   +------------------+---------------+--------------------+
+   | Data             | 4D 61 69 6C   | "Mail"             |
+   +------------------+---------------+--------------------+
+
+#. Press **Button 2** to request app attributes for the app with the app identifier "com" (the last received app identifier).
+#. In nRF Connect, verify that the :guilabel:`Apple Control Point` is updated to ``01 63 6F 6D 00 00``.
+#. Respond to the request by sending the app attribute.
+   Set the :guilabel:`Apple Data Source` value in the Server to ``01 63 6F 6D 00 00 04 00 4D 61 69 6C``.
+#. The application will print the received data on UART. Verify that the UART output is as follows::
+
+      Display Name: Mail
+
+Disconnect
+^^^^^^^^^^
+
+#. Disconnect the device in nRF Connect.
```

Comment:
I wanted to move the numbered list. To me, this does not seem to be two separate steps. The second sentence just states how easily you can reconnect as nRF Connect still has the bond information.

Reply:
Done